DICOM/src/Microsoft.Health.DeID.SharedLib/Models/AgeObject.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib.Models
+{
+    public class AgeObject
+    {
+        public AgeObject(uint age, AgeType ageType)
+        {
+            Value = age;
+            AgeType = ageType;
+        }
+
+        public uint Value { get; } = 0;
+
+        public AgeType AgeType { get; }
+
+        public decimal AgeInYears()
```

Comment:
🟡 **SUGGESTION**
Why is the return type `decimal` ?
It's calculated by `Value / 12`, the result is still an integer.
If the result value will be used as decimal somewhere, like `RedactAge`, it can be simply casted.

Reply:
Changed it to uint.